sunpy/map/sources/soho.py

```diff
@@ -123,6 +124,17 @@ def __init__(self, data, header, **kwargs):
         self.plot_settings['norm'] = ImageNormalize(
             stretch=source_stretch(self.meta, PowerStretch(0.5)), clip=False)
 
+        # For Helioviewer images, clear rotation metadata, as these have already been rotated.
+        # Also check that all CROTAn keywords exist to make sure that it's an untouched
+        # Helioviewer file.
+        if 'helioviewer' in self.meta \
+                and 'crota' in self.meta and 'crota1' in self.meta and 'crota2' in self.meta:
```

Comment:
I have a hatred of line continuation characters :stuck_out_tongue: 
```suggestion
        if ('helioviewer' in self.meta 
                and 'crota' in self.meta and 'crota1' in self.meta and 'crota2' in self.meta):
```

Reply:
Haha, no problem 😅 Replaced in f6496e2.